src/components/PortfolioCard/PortfolioCardTablet.tsx

```diff
@@ -2,8 +2,13 @@ import React from 'react';
 import { Smartphone } from 'react-feather';
 import { Card, CardActionArea, CardContent, CardMedia, makeStyles, Typography } from '@material-ui/core';
 
+import { transformImage } from '@/utils/image';
+
 import type { PortfolioCardVariantProps } from './PortfolioCard.types';
 
+const PROJECT_IMAGE_WIDTH = 240;
```

Comment:
I'm not the biggest fan of declaring these two variables as constants, I think that it made sense when the values were the same for both height and width but here is seems redundant to me, wdyt?

Reply:
Those values need to be the same in the component and its styles, so this way we can ensure they always are

Reply:
Makes sense 👍